client/web/src/enterprise/batches/detail/BatchChangeTabs.tsx

```diff
@@ -123,7 +127,8 @@ export const BatchChangeTabs: React.FunctionComponent<BatchChangeTabsProps> = ({
                             onClick={onSelectChangesets}
                             className={classNames('nav-link', selectedTab === 'changesets' && 'active')}
                         >
-                            <SourceBranchIcon className="icon-inline text-muted mr-1" /> Changesets
+                            <SourceBranchIcon className="icon-inline text-muted mr-1" />{' '}
+                            {changesetsCount === 1 ? '1 Changeset' : `${changesetsCount} Changesets`}
```

Comment:
```suggestion
                            <SourceBranchIcon className="icon-inline text-muted mr-1" />{' '}
                            {changesetsCount} {pluralize('Changeset', changesetsCount)}
```

we got a little helper for that :)